helm/oncall/templates/_env.tpl

```diff
@@ -517,15 +553,43 @@
 {{- end }}
 
 {{- define "snippet.redis.env" -}}
+- name: REDIS_PROTOCOL
+  value: {{ include "snippet.redis.protocol" . }}
 - name: REDIS_HOST
   value: {{ include "snippet.redis.host" . }}
 - name: REDIS_PORT
-  value: "6379"
+  value: {{ include "snippet.redis.port" . }}
+- name: REDIS_DATABASE
+  value: {{ include "snippet.redis.database" . }}
+{{- with .Values.externalRedis.username }}
+- name: REDIS_USERNAME
+  value: {{ . | quote }}
+{{- end }}
```

Comment:
I hid it because it is another rarely used environment variable in the deployment manifest. With default `""` value will work as expected. I'm preparing the fix. Thanks :)